src/domains/sign/integration.tf

```diff
@@ -0,0 +1,121 @@
+module "event_hub" {
+  source                   = "git::https://github.com/pagopa/terraform-azurerm-v3.git//eventhub?ref=v4.1.7"
+  name                     = format("%s-eventhub-ns", local.project)
+  location                 = azurerm_resource_group.integration_rg.location
+  resource_group_name      = azurerm_resource_group.integration_rg.name
+  auto_inflate_enabled     = var.integration_hub.auto_inflate_enabled
+  sku                      = var.integration_hub.sku_name
+  capacity                 = var.integration_hub.capacity
+  maximum_throughput_units = var.integration_hub.maximum_throughput_units
+  zone_redundant           = var.integration_hub.zone_redundant
+
+  virtual_network_ids = [data.azurerm_virtual_network.vnet_common.id]
+  subnet_id           = module.io_sign_eventhub_snet.id
+
+  private_dns_zone_record_A_name = null
+
+  eventhubs = var.integration_hub.hubs
+
+  public_network_access_enabled = true
+  network_rulesets = [
+    {
+      default_action = "Deny",
+      virtual_network_rule = [
+        {
+          subnet_id                                       = module.io_sign_eventhub_snet.id,
+          ignore_missing_virtual_network_service_endpoint = false
+        }
+      ],
+      ip_rule = var.integration_hub.ip_rules
+    }
+  ]
+
+  private_dns_zones = {
+    id   = [data.azurerm_private_dns_zone.privatelink_servicebus_windows_net.id]
+    name = [data.azurerm_private_dns_zone.privatelink_servicebus_windows_net.name]
+  }
+
+  alerts_enabled = var.integration_hub.alerts_enabled
+  metric_alerts = {
+    no_trx = {
+      aggregation = "Total"
+      metric_name = "IncomingMessages"
+      description = "No transactions received from acquirer in the last 24h"
+      operator    = "LessThanOrEqual"
+      threshold   = 1000
+      frequency   = "PT1H"
+      window_size = "P1D"
+      dimension = [
+        {
+          name     = "EntityName"
+          operator = "Include"
+          values   = ["rtd-trx"]
+        }
+      ],
+    },
+    active_connections = {
+      aggregation = "Average"
+      metric_name = "ActiveConnections"
+      description = null
+      operator    = "LessThanOrEqual"
+      threshold   = 0
+      frequency   = "PT5M"
+      window_size = "PT15M"
+      dimension   = [],
+    },
+    error_trx = {
+      aggregation = "Total"
+      metric_name = "IncomingMessages"
+      description = "Transactions rejected from one acquirer file received. trx write on eventhub. check immediately"
```

Comment:
ditto

Reply:
Acquirer is a terminology used in the business domain of the integrated solution and approved by a design review. Both its meaning and usage are documented in the internal company wiki.